samples/subsys/usb/hid/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _usb_hid:
+
+USB HID Sample Application
+##########################
+
+Overview
+********
+
+This sample app demonstrates use of a USB Human Interface Device (HID) driver
+by the Zephyr project.  This very simple driver is enumerated as a raw HID
+device. This sample can be found under :file:`samples/subsys/usb/hid` in the
+Zephyr project tree.
+
+Requirements
+************
+
+This project requires an USB device driver.
```

Comment:
Sorry I missed this earlier.  Change “an USB” to “a USB”

Reply:
@dbkinder Could you recheck?